src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLabeledBreak.java

```diff
@@ -0,0 +1,60 @@
+/*
+FallThrough
+checkLastCaseGroup = (default)false
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThroughLabeledBreak {
```

Comment:
Please add a couple more test cases:
1. break to outer labelled block, like:
```
outer: { switch(a) {case 1: break outer; ... }}
```
2. Break to label with comment, like `break /* comment */ top`

Reply:
done